ngraph/core/include/ngraph/op/fake_quantize.hpp

```diff
@@ -67,6 +67,13 @@ class NGRAPH_API FakeQuantize : public ngraph::op::Op {
         m_auto_broadcast = auto_broadcast;
     }
 
+    bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
```

Comment:
Was there any particular reason to add the evaluate method to this op? It's already registered in https://github.com/openvinotoolkit/openvino/blob/master/ngraph/test/runtime/interpreter/evaluates_map.cpp#L2437-L2457 so it should be available from the interpreter backend if this was the case.

Reply:
CPU plugin does not support a fake quantize where range is not constant.